examples/golang/object-upload/README.md

```diff
@@ -11,6 +11,6 @@ Go installed on the Linux machine.
 ```
 aws --endpoint-url http://localhost:8000 s3 mb s3://sample-bucket
 ```
-3. Navigate through your file system to where the Go example code is using your terminal.
+3. Navigate through your file system to where the Golang example code exist on your terminal.
```

Comment:
In a terminal window, cd into the directory with the example Go code.

Reply:
> In a terminal window, cd into the directory with the example Go code.

thanks! will be fixed in another PR

Reply:
Groovy.  I knew what was being said here, but the language is really awkward.